### PR TITLE
Code Quality: Update UWPToWinAppSDKUpgradeHelpers

### PR DESCRIPTION
**Pull request by 0x5bfa**
### Motivation and Context

This file was for helping developers who wanted to migrate UWP to WASDK. But actual function is Interop PInvoke service.

### PR Checklist
- [ ] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
- [ ] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [ ] Are there any other steps that were used to validate these changes?

### Screenshots

_None_

### CODE REVIEW

**Comment by 0x5bfa** [comment]
Ready for review.

File: src/Files.App/Helpers/InteropHelpers.cs [hunk 1]
```diff
@@ -3,59 +3,60 @@
...
+
+			type.InvokeMember(
+				"ProtectedCursor",
+				BindingFlags.Public | BindingFlags.NonPublic | BindingFlags.SetProperty | BindingFlags.Instance,
+				null,
+				uiElement,
+				new object[] { cursor }
+			);
 		}
 	}
```
Comment (hez2010) [inline src/Files.App/Helpers/InteropHelpers.cs:51]: Should we get rid of this reflection call and change it to call ProtectedCursor setter from UIElement derived types directly instead?

Comment (0x5bfa) [reply src/Files.App/Helpers/InteropHelpers.cs:51]: Might as well avoid unrelated discussion.

Comment (hez2010) [reply src/Files.App/Helpers/InteropHelpers.cs:51]: Yes. That's why I didn't leave this as a code review.

File: src/Files.App/Helpers/InteropHelpers.cs [hunk 2]
```diff
@@ -3,59 +3,60 @@
...
-		IntPtr GetForWindow([In] IntPtr appWindow, [In] ref Guid riid);
-
-		void ShowShareUIForWindow(IntPtr appWindow);
-	}
-
 	public static class InteropHelpers
 	{
 		public static readonly Guid DataTransferManagerInteropIID =
-			new Guid(0xa5caee9b, 0x8708, 0x49d1, 0x8d, 0x36, 0x67, 0xd2, 0x5a, 0x8d, 0xa0, 0x0c);
+			new(0xa5caee9b, 0x8708, 0x49d1, 0x8d, 0x36, 0x67, 0xd2, 0x5a, 0x8d, 0xa0, 0x0c);
```
Comment (yaira2) [inline src/Files.App/Helpers/InteropHelpers.cs:14]: Let's keep `Guid`

Comment (0x5bfa) [reply src/Files.App/Helpers/InteropHelpers.cs:14]: That can be simplified. however?

Comment (yaira2) [reply src/Files.App/Helpers/InteropHelpers.cs:14]: We usually only simplify it if it's already clear what the type is. I just noticed that it's defined on line 13 so disregard what I said.

**Review by yaira2 — APPROVED** [review]
LGTM

**Pull request merged**